db_import/import_db.sh

```diff
@@ -1,32 +1,29 @@
 #!/bin/bash
```

Comment:
Modified script assuming most of people will have a default local mongo installation